torch/_dynamo/compiled_autograd.py

```diff
@@ -168,18 +168,28 @@ def bind_tensors_to_proxies(self, tensors, proxies):
         track_tensor_tree(tensors, proxies, constant=None, tracer=self.fx_tracer)
 
 
+compiled_autograd_enabled = False
```

Comment:
This feels a little gross, but I wasn't sure of a better way to do it with the `prior` pattern below.

Reply:
I would naively have expected to have a way to `get_autograd_compiler` and then I would simply test if that is None or not. Does this API not exist?

Reply:
I did not find it, just the set. It writes to a value in C++. It seemed like a bigger API to expose for little gain atm. If we want, we can always expose that later.

Reply:
This is single source of truth principle: it's better to have one C++ variable with a Python accessor, rather than two sources of truth that can desync. I think that's worth it, the getter is very simple